Jenkinsfile

```diff
@@ -86,16 +86,75 @@ try {
         }
     }
 
+    tasks['runPylint'] = {
+        stage('runPylint')
+        {
+            node(Constants.DOCKERNODE) {
+                withContainer(image: "python:${version}-buster", registry: '', inside: '-u root') {
+                    try {
+                        sh """
+                           pylint --exit-zero --output-format=parseable --reports=n tenable > reports/pylint_tenable.log
+                           pylint --exit-zero --output-format=parseable --reports=n tests > reports/pylint_tests.log
+                           cat reports/pylint_tenable.log
+                           cat reports/pylint_tests.log
+                        """
+                    } catch(ex){
+                        throw ex
+                    } finally {
+                        if (fileExists ('reports/pylint_tenable.log')) {
+                               //result needs to outpyt, will test
+                               result =  recordIssues(
+                                    enabledForFailure: true,
+                                    tool: pyLint(pattern: 'reports/pylint_tenable.log'),
+                                    unstableTotalAll: 20,
+                                    failedTotalAll: 30,)
+                                                                        }
```

Comment:
Formatting is bad - most closing brackets are not in line with the code. Makes the file hard to read.